src/main/main.ts

```diff
@@ -40,17 +40,28 @@ const main = async () => {
     // avoid white border on top frame
     // https://github.com/electron/electron/issues/13164#issuecomment-479941434
     titleBarStyle: 'customButtonsOnHover',
+    // required on fullscreen
+    show: false,
     // cannot exit by cmd + q
     // closable: false,
   })
 
+  // on fullscreen hack for macos
+  mainWindow.on('ready-to-show', () => {
+    app.dock && app.dock.hide()
+    mainWindow.show()
+    app.dock && app.dock.show()
+  })
```

Comment:
Refs. https://github.com/codyseibert/youtube/blob/7afc1b97d5dca26c75021fedf4a883a9880389b9/electron-in-5-minutes/main.js